bgpd/bgp_vty.c

```diff
@@ -17382,6 +17400,9 @@ int bgp_config_write(struct vty *vty)
 		if (CHECK_FLAG(bgp->flags, BGP_FLAG_SHUTDOWN))
 			vty_out(vty, " bgp shutdown\n");
 
+		if (bgp->allow_martian)
+			vty_out(vty, " bgp allow-martian\n");
```

Comment:
This should be changed to `allow-martian-nexthop` as well?

Reply:
fixed